### DISCUSSION TITLE
How to select an object by a unique attribute?

### DISCUSSION THREAD

**Discussion by festing** [olton/Metro-UI-CSS#1933] [General]
Certain divs have a unique attribute "bmIFD".  How do I select all object with this unique attribute?

**Comment by olton** [comment]
html example please

**Comment by festing** [comment]
```
<div class=" border-top bd-darkOlive shadow-1 p-1" id="3470D64B-CD89-725E-E219F5F9CED7DED1" bmIFD="2065ef15-fbb2-83b5-49ac-3eced3a8aba8" bmid="0" data-role-touch="true" data-role="touch">
 HTML STUFF
</div>

```
I'll have multiple DIVS with that same bmIFD.  All I'm trying to do is change bkg color of JUST those elements

**Comment by olton** [comment]
`$("[bmIFD]")` - select all with attribute
`$("[bmIFD=2065ef15-fbb2-83b5-49ac-3eced3a8aba8]")` - select all with attribute with value

**Comment by festing** [comment]
Almost there.  How do I make value a variable:  IE: 
var eventID = $(this).attr("bmIFD")
$("[bmIFD=eventID]").addClass("bg-lime")

**Comment by olton** [comment]
use backward quotes "`"
```
var eventID = $(this).attr("bmIFD")
$(`[bmIFD=${eventID}]`).addClass("bg-lime")
```

**Reply by festing** [reply]
So that should addClass to ALL divs with that "bmIFD" value yes?  Does not seem to work

**Reply by festing** [reply]
Any idea why this doesn't work?  Seems like it should
